_data/projects/docusaurus-json-schema-plugin.yml

```diff
@@ -0,0 +1,10 @@
+name: Docusaurus JSON Schema Plugin
+desc: JSON Schema plugin viewer in Docusaurus
+site: https://jy95.github.io/docusaurus-json-schema-plugin/
+tags:
+  - json
+  - schema
+  - docusaurus
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/jy95/docusaurus-json-schema-plugin/discussions/categories/ideas
```

Comment:
We don't currently have support for a discussion category, but this is interesting to potentially support. Would need some plumbing changes to do a different query to compute the count...

Reply:
I found the "top" sorting interesting for better community engagement